apps/frontend/src/libs/types/input-options.type.ts

```diff
@@ -0,0 +1,6 @@
+type Options = {
```

Comment:
```suggestion
type RadioInputOption = {
```
Change the file name as well